src/rust/plugin-registry/src/static_files/plugin.nomad

```diff
@@ -30,6 +34,16 @@ variable "aws_account_id" {
   description = "The account ID of the aws account that holds onto the plugin binaries."
 }
 
+variable "plugin_bootstrap_container_image" {
+  type        = string
+  description = "The tenant-plugin-bootstrap-sidecar task's DockerImageId."
+}
+
+variable "plugin_execution_container_image" {
+  type        = string
+  description = "The tenant-plugin-execution-sidecar task's DockerImageId."
```

Comment:
Terminology nitpick: I would call these container image _names_... an ID is something like `0e5574283393`

Reply:
Unfortunately I use this terminology everywhere - can I change it in an unrelated PR?

```

"""
A Docker image identifier is something that can be consumed by the
Nomad Docker plugin `image` field.
https://www.nomadproject.io/docs/drivers/docker#image
The values can look like, for instance:
- a hardcoded value pulled from Dockerhub
    "dgraph/dgraph:v21.0.3"
- an image pulled from the host's Docker daemon (no `:latest`!)
    "model-plugin-deployer:dev"
- an image pulled from Cloudsmith
    "docker.cloudsmith.io/grapl/raw/graph-merger:20211105192234-a86a8ad2"
"""
DockerImageId = NewType("DockerImageId", str)
```

Reply:
Yeah, that's fine